packages/manager/tools/testcafe/pages/manager/components/navbar.js

```diff
@@ -0,0 +1,9 @@
+import { Selector } from 'testcafe';
+
+class Navbar {
+  constructor() {
+    this.navbar = Selector('.oui-navbar');
```

Comment:
We should try to be less dependent on classes and ids and have specific selector for tests

Reply:
I agree, here it's just an example, that act as a proof of concept. We can then put a better selector